packages/path_provider/path_provider_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## NEXT
```

Comment:
You should go ahead and version bump this, since plugin clients using newer SDK versions will hit the warning as well; we get issues filed about build warnings, so people do notice.